### PR TITLE
Run acceptance tests on Travis.

### PR DESCRIPTION
**Pull request by 23Skidoo**

### CODE REVIEW

**Comment by 23Skidoo** [comment]
@tnaeg This adds `genomizer-server-tester` (with my tweaks) to the `genomizer-server` repo, so you'll probably want to delete [that repo](https://github.com/genomizer/genomizer-server-tester) once this is merged.

**Comment by 23Skidoo** [comment]
Finally got it to pass!

File: genomizer-server-tester/src/api/commands/GenomeRelease/GetSpecificGenomeTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    @Override
+    public void execute() {
+        try {
+            CommandTester.conn.sendRequest(new GetGenomeSpecieReleasesRequest(specie),
+                    CommandTester.token, Constants.TEXT_PLAIN);
+
+            if (CommandTester.conn.getResponseCode() == 200) {
+                String respBody = CommandTester.conn.getResponseBody();
+                for (String expectedStr : expected)
+                    super.finalResult = respBody.contains(expectedStr);
```
Comment (Mintey) [inline genomizer-server-tester/src/api/commands/GenomeRelease/GetSpecificGenomeTest.java]: I assume that this is not the wanted functionality? 
`super.finalResult` only seems to depend on the last item in the array, not all of them, which makes changing expected to an array unneccary.

**Comment by Mintey** [comment]
Other than the above comment, it looks like it'll work just fine on travis and still be able to connect to the vagrant servers via the 700X ports. Good job making it work on travis, me and @tnaeg did not think that would be possible.

File: genomizer-server-tester/src/api/commands/GenomeRelease/GetSpecificGenomeTest.java (hunk 1 above)
Comment (23Skidoo) [inline genomizer-server-tester/src/api/commands/GenomeRelease/GetSpecificGenomeTest.java]: Good catch. Yes, it should be `finalResult = finalResult && someTest()`.

**Pull request merged**